Hmm, I just tried to build and got:

```
Traceback (most recent call last):
  File "/home/dan/src/kolibri/kolibri-installer-android/scripts/version.py", line 143, in <module>
    print(get_ek_version())
  File "/home/dan/src/kolibri/kolibri-installer-android/scripts/version.py", line 115, in get_ek_version
    explore_plugin_version(), kolibri_version(), android_version_indicator
  File "/home/dan/src/kolibri/kolibri-installer-android/scripts/version.py", line 40, in explore_plugin_version
    spec = importlib.util.spec_from_file_location(
AttributeError: module 'importlib' has no attribute 'util'
```

So, uh, hang on.

--- a/scripts/version.py
+++ b/scripts/version.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-import importlib
+import importlib.util
 import os
 import subprocess
 import sys

